ex5.1_this_is_the_way.py

```diff
@@ -0,0 +1,12 @@
+import os
+INPUT_FOLDER_MESSAGE="insert the path to the folder:"
+# function that print the file in folder that start with "deep"
```

Comment:
![Capture](https://user-images.githubusercontent.com/13467749/159539679-2db1201c-867e-464d-bec4-7f860f4722e5.JPG)
that's the standard format of python documentation  - use it